code/__DEFINES/is_helpers.dm

```diff
@@ -152,11 +151,10 @@ GLOBAL_LIST_INIT(shoefootmob, typecacheof(list(
 	/mob/living/simple_animal/hostile/retaliate/spaceman,
 	/mob/living/simple_animal/hostile/retaliate/nanotrasenpeace,
 	/mob/living/simple_animal/hostile/retaliate/goat,
-	/mob/living/carbon/true_devil,
+	/mob/living/carbon/true_devil
 	)))
 
 GLOBAL_LIST_INIT(clawfootmob, typecacheof(list(
-	/mob/living/carbon/alien/humanoid,
```

Comment:
Before anyone asks about these changes, clawfoot and shoefoot are literally only used for footsteps.